samples/package.json

```diff
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "@google-cloud/storage": "^1.5.2",
-    "@google-cloud/vision": "0.19.0",
+    "@google-cloud/vision": "0.21.0",
```

Comment:
0.20.0!

Reply:
yep. fixing this now. something odd happened